x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_key_flyout.tsx

```diff
@@ -55,6 +58,22 @@ import type {
   UpdateAPIKeyResult,
 } from '../api_keys_api_client';
 
+export const DEFAULT_EXPIRATION_VALUE = '60';
+
+export const ENABLED_LABEL: string = i18n.translate(
+  'xpack.security.accountManagement.apiKeyFlyout.enabledLabel',
+  {
+    defaultMessage: 'Enabled',
+  }
+);
+
+export const DISABLED_LABEL: string = i18n.translate(
+  'xpack.security.accountManagement.apiKeyFlyout.disabledLabel',
+  {
+    defaultMessage: 'Disabled',
+  }
+);
+
```

Comment:
Nit: These 3 constants appear to be unused

Reply:
Removed :)